tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./App.{ts,tsx}', './src/**/*.{ts,tsx}'],
+  theme: {
+    extend: { colors: { verde: '#3BBB30', roxo: '#1967FB', roxoP: '#6448B7' } },
```

Comment:
Alterar nome das cores
```suggestion
    extend: { colors: { verde: '#3BBB30', roxoClaro: '#1967FB', roxoEscuro: '#6448B7' } },
```